### DISCUSSION TITLE
AraGPT2 & AraELECTRA

### DISCUSSION THREAD

**Discussion by israakhalil** [aub-mind/arabert#107] [Q&A]
Hello,
Can I use AraGPT2 & AraELECTRA for text Classification  ?

**Comment by WissamAntoun** [comment]
Yes of course, But i don't think AraGPT2 will perform well

**Reply by israakhalil** [reply]
How can I use them
Is it the same as using AraBert in this example https://colab.research.google.com/drive/1P9iQHtUH5KUbTVtp8B4-AopZzEEPE0lw?usp=sharing
But with changing the name of the model?

**Reply by WissamAntoun** [reply]
This notebook will work for AraELECTRA and not for AraGPT2. For araGPT2 you will have to edit the BERTdataset class, and remove the [CLS] token.
